symbols-mui/scripts/linkComponents.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+BLUE='\033[0;34m'
+BBLUE='\033[1;34m' #BOLD
+PURPLE='\033[0;35m'
+RED='\033[0;31m'
+BRED='\033[1;31m' #BOLD
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m' #BOLD
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Building components...${NC}"
+yarn build
+
+echo -en "${BLUE}Creating new folder in node_modules...${NC}"
+rm -rf "./demos/showcase/node_modules/@pxblue/symbols-mui"
+mkdir -p "./demos/showcase/node_modules/@pxblue/symbols-mui"
+# rm -rf "./demos/storybook/node_modules/@pxblue/symbols-mui"
+# mkdir -p "./demos/storybook/node_modules/@pxblue/symbols-mui"
+echo -e "${GREEN}Done${NC}"
+
+echo -en "${BLUE}Copying build output into node_modules...${NC}";
+# cp -r ./symbols/package.json ./demos/showcase/node_modules/@pxblue/symbols-mui/package.json
+cp -r ./symbols/. ./demos/showcase/node_modules/@pxblue/symbols-mui
+# cp -r ./components/package.json ./demos/storybook/node_modules/@pxblue/symbols-mui/package.json
+# cp -r ./dist/. ./demos/storybook/node_modules/@pxblue/symbols-mui
+echo -e "${GREEN}Done${NC}"
+
+echo -en "\r\n${BLUE}Linking Components: ${NC}"
+if [ ! -f ./demos/showcase/node_modules/@pxblue/symbols-mui/package.json ]; then echo -e "${BRED}Not Linked${NC}" && exit 1; fi
+# if [ ! -f ./demos/storybook/node_modules/@pxblue/symbols-mui/package.json ]; then echo -e "${BRED}Not Linked${NC}" && exit 1; fi
```

Comment:
is this a duplicate of the line above? Can we get rid of these comments or do we need them?

Reply:
I've removed all the comments, not sure how long they've been there for.